### ISSUE TITLE
Cuttlefish is holding back comment report emails

### ISSUE THREAD

**Issue by henare** [openaustralia/planningalerts#688]

**Comment by henare** [comment]
Someone with an `@yahoo.com` address reported a comment and Google responded to the report email saying:

```
bounced (host aspmx.l.google.com[74.125.20.27] said: 550-5.7.1 Unauthenticated email from yahoo.com is not accepted due to domain's 550-5.7.1 DMARC policy. P... 
```

So the address got added to the blacklist.

I've removed it and opened #689 to track the underlying problem.

**Comment by equivalentideas** [comment]
This happened again 9 days ago https://cuttlefish.oaf.org.au/emails/3502465

Reports have been rejected since.

I've removed contact@planningalerts.org.au from the blacklist and am sending a test report.

**Comment by equivalentideas** [comment]
Not sure if this is the same problem as #689 so I'm reopening this.

**Comment by henare** [comment]
It got blacklisted again. This time for the same reason as https://github.com/openaustralia/righttoknow/issues/551 - a yahoo user reported a comment.

**Comment by equivalentideas** [comment]
It got blacklisted again, 6 months ago at least. I just discovered this :(

**Comment by benrfairless** [comment]
How about changing the "from" address to noreply@planningalerts.com.au and setting the "reply to" address as the user's email address. That's how Right to Know works for its contact forms now.

**Comment by equivalentideas** [comment]
Clearing this out now. It's been blacklisted since 2016-07-20 10:53:45 😢

**Comment by equivalentideas** [comment]
> How about changing the "from" address to noreply@planningalerts.com.au and setting the "reply to" address as the user's email address. That's how Right to Know works for its contact forms now.

@benrfairless I don't understand enough about how blacklisting works to know if that would fix this, but sounds like that could be good 👍

**Comment by henare** [comment]
>How about changing the "from" address...

@equivalentideas @benrfairless I think that's what's suggested in #689.

**Comment by henare** [comment]
This should be fixed in #1151 so I've removed the address from the blacklist.
